### ISSUE TITLE
Manual approval not working

### ISSUE THREAD

**Issue by errogaht** [jenkinsci/telegram-notifications-plugin#39]
no users in list 
![image](https://user-images.githubusercontent.com/6324265/50002219-6a724200-ffb0-11e8-89a9-018b360a158d.png)
just auto aproval working.
/sub command sent sucessfully, bot said "wait for approval from admin"

**Comment by maxiv** [comment]
Same problem.

**Comment by RandomReaper** [comment]
is that related to username being empty in telegram? this field is not mandatory in telegram

**Comment by baldrs** [comment]
@RandomReaper nope I have it filled, still it doesn't work.

**Comment by baldrs** [comment]
Current workaround seems to be quickly turn it on and off, however this offers zero security and protection because you don't actually know who had subscribed.

**Comment by Brother-Andy** [comment]
@baldrs 

I faced same bug and here is my workaround:
1. Add bot settings and select manual approve.
2. Go to /var/lib/jenkins/jenkinsci.plugins.telegrambot.TelegramBotGlobalConfiguration.xml
3. Find user section  at 85   <users>
4. Change <isApproved>false</isApproved> to <isApproved>true</isApproved>
5. Restart Jenkins.

After these actions user is able to receive notifications.
However, approved user still is not present in the list of subscribed accounts at jenkins setting page.

**Comment by maxiv** [comment]
@baldrs
I haven't user data in <users/> section.

**Comment by maxiv** [comment]
I think problem is in saving config file, when webhook get message from Telegram API. In manual mode I send /sub and /status message, and got answer that I already subscribed, than I do Jenkins restart and /status return that I not subscribed.

**Comment by rpanak-generalbytes** [comment]
@Brother-Andy Can you please post a sample <users> section from the config file? The users saving seems to be broken completely. I'm trying to work around the problem and configure it manually in the XML file. A sample with correct syntax would help.

**Comment by rpanak-generalbytes** [comment]
Never mind, I've managed to get a config sample.

**Comment by maxiv** [comment]
> Never mind, I've managed to get a config sample.

How do you do that?

**Comment by rpanak-generalbytes** [comment]
> How do you do that?

I've fixed some errors in the code and could update the config file. Later I've decided to dig a little deeper and I fixed these problems:
- subscriber list not updating for manual approval (the main problem)
- log to console always checked on the settings page
- if the subscribing user has no telegram username, the subscriber list is not really helpful.

I don't understand why hasn't anybody fixed this yet. Those were easy fixes. It seems it was some sloppy refactoring that introduced the bugs. I have to do some test first and I might have a PR ready soon.
